pages/api/add-user.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ResponseType } from '../../interfaces';
+import { prisma } from '../../lib/prisma';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<ResponseType>
+) {
+  const { email } = req.body;
+
+  if (req.method === 'POST') {
+    const new_user = await prisma.user
```

Comment:
make the variable name newUser. please follow camel case in variables. in all places please refactor it to camel case